templates/website-footer.md

```diff
@@ -5,8 +5,25 @@ status: draft
 
 For those building or managing a website for a Commonhaus Foundation project, please incorporate one of the following standard footers, which have been vetted and approved by the Commonhaus Foundation legal counsel:
 
-## TBD: Example text
+## HTML
 
-> Copyright © [Commonhaus Foundation](https://website) and [project name] contributors. All rights reserved. The [Commonhaus Foundation](https://website) has registered trademarks and uses trademarks. For a comprehensive list of Commonhaus Foundation trademarks, consult our [Trademark Policy](https://trademark-policy.website/) and [Trademark List](https://trademark-list.website/). Any trademarks or logos not listed on the [Commonhaus Foundation trademarks list](https://trademark-list.website) are trademarks™ or registered® trademarks owned by their respective holders. Their use does not suggest any affiliation with or endorsement by them.
+```html
+<p>
+  Copyright © <a href="https://www.commonhaus.org">Commonhaus Foundation</a>. All rights reserved. For details on our trademarks, please visit our <a href="/policies/trademark-policy/">Trademark Policy</a> and <a href="/trademarks/">Trademark List</a>. Trademarks of third parties are owned by their respective holders and their mention here does not suggest any endorsement or association.<br />
```

Comment:
Should the trademark policy and list links be the full href like they are below?

Reply:
oh. oops. yes.